cpp/src/arrow/csv/options.h

```diff
@@ -24,6 +24,7 @@
 #include <unordered_map>
 #include <vector>
 
+#include "arrow/timestamp_converter.h"
```

Comment:
```suggestion
```

Reply:
We need to decide how to deal with interfaces before.

Reply:
Use a forward declaration if you can